apiserver/uniter/uniter_test.go

```diff
@@ -2755,11 +2763,391 @@ func (s *uniterNetworkConfigSuite) TestNetworkConfigForImplicitlyBoundEndpoint(c
 
 	expectedConfig := []params.NetworkConfig{{Address: privateAddress.Value}}
 
-	result, err := s.base.uniter.NetworkConfig(args)
+	result, err := s.base.uniterv3.NetworkConfig(args)
 	c.Assert(err, jc.ErrorIsNil)
 	c.Assert(result, jc.DeepEquals, params.UnitNetworkConfigResults{
 		Results: []params.UnitNetworkConfigResult{
 			{Config: expectedConfig},
 		},
 	})
 }
+
+type uniterNetworkInfoSuite struct {
+	base uniterSuite
```

Comment:
It's completely based on NetworkConfig suite, I'll add a ticket to clean up the whole uniter_test.